data-and-functions-1/src/functions/getProductById.js

```diff
@@ -0,0 +1,11 @@
+const getProductById = (data, id) => {
+  if (data == null || data.products == null || id == null) {
+    return null
+  } else {
+    return data.products.find((p) => {

```

Comment:
Nice use of `find`. You can drop the block and use an implicit return if you want. You can also drop the parens around `(p)` since you only have one param in your lambda.